static/bidder-params/adagio.json

```diff
@@ -50,7 +50,7 @@
         "^[a-zA-Z_]": { "type": "string" }
       }
     },
-    "prebidVersion:": {
```

Comment:
Can you please add tests for this and other parameters supported by your adapter in `adagio/params_test.json`. Currently, it only covers `organizationId`, `site` and, `placement` parameters? Adding tests for these other parameters will help catch these errors sooner.